pyomo/core/base/initializer.py

```diff
@@ -111,7 +126,13 @@ def Initializer(init,
         # generator into a tuple and then store it as a constant.
         return ConstantInitializer(tuple(init))
     if type(init) is functools.partial:
-        _args = inspect.getfullargspec(init.func)
+        try:
+            _args = inspect.getfullargspec(init.func)
+        except:
+            # Inspect doesn't work for some built-in callables (notably
```

Comment:
Does one of the new tests check against this assumption?

Reply:
Yes: test_initializer.py:531 (`test_no_argspec`)